tests/group_by_test.py

```diff
@@ -118,8 +119,22 @@ def test_group_by_depth_1_agg(
 ) -> None:
     if "cudf" in str(constructor) and attr == "n_unique":
         request.applymarker(pytest.mark.xfail)
+    if (
+        "pandas" in str(constructor)
+        and attr == "var"
+        and PANDAS_VERSION < (2, 0, 9)
+        and "pyarrow" in str(constructor)
+    ):
+        request.applymarker(
+            pytest.mark.xfail(
+                reason="Known issue with variance calculation in pandas 2.0.x with pyarrow backend in groupby operations"
+            )
+        )
     data = {"a": [1, 1, 1, 2], "b": [1, None, 2, 3]}
-    expr = getattr(nw.col("b"), attr)()
+    if attr in ["var", "std"]:
+        expr = getattr(nw.col("b"), attr)(ddof=1)
+    else:
+        expr = getattr(nw.col("b"), attr)()
```

Comment:
this doesn't need changing, `ddof=1` is already the public-facing default, it's only in the internal code that we tend to not set defaults
```suggestion
    if attr in ["var", "std"]:
        expr = getattr(nw.col("b"), attr)(ddof=1)
    else:
        expr = getattr(nw.col("b"), attr)()
```